### PR TITLE
Support Lucee logger config

### PR DESCRIPTION
**Pull request by jcberquist**
Adds support for the lucee logger configuration. 

One possible thing that should still be addressed is URL encoding the settings. I have noticed that the default settings for the loggers in Lucee are not URL encoded, but if you make changes they do end up URL encoded.

### CODE REVIEW

File: models/BaseLucee.cfc [hunk 1]
```diff
@@ -803,6 +826,47 @@ component accessors=true extends='cfconfig-services.models.BaseConfig' {
...
+			// Create new logger tag
+			} else {
+				var loggerXMLNode = xmlElemnew( thisConfig, "logger" );
+			}
+
+			// Populate XML node
+			loggerXMLNode.XMLAttributes[ 'name' ] = name;
+			for ( var key in [ 'appender', 'appenderClass', 'layout', 'layoutClass', 'level' ] ) {
+				if ( !isNull( loggerStruct[ key ] ) ) { 
+					loggerXMLNode.XMLAttributes[ key.replace( 'C', '-c' ) ] = loggerStruct[ key ];
```
Comment (bdw429s) [inline models/BaseLucee.cfc]: @jcberquist What's this replace doing? Can you add a comment in the code?

**Comment by bdw429s** [comment]
Thx @jcberquist this looks pretty good. If you can add a comment to explain those two lines that doesn't make sense on their own, I'll merge it.  Do you want to take a stab at adding CLI commands for `cfconfig logger list`, cfconfig logger save`, and `cfconfig logger delete`?  That would go to the https://github.com/Ortus-Solutions/commandbox-cfconfig repo.

**Comment by jcberquist** [comment]
I will take a look at adding the commandbox commands too. Thanks!

**Comment by bdw429s** [comment]
Thanks for your work on this @jcberquist !  I'll bump a new version right now so people can give it a test.  

I'd love to see a blog post or something if you have the time to encourage that shows what you did so other people can dive in a contribute to CFConfig.  There will be a lot of edge-case settings that specific people will want done and I want it to be easy for them to help add them.  Also, please let me know if I can do anything to help contributors find their way around.

**Comment by bdw429s** [comment]
New CFConfig version `0.17.0-alpha` pubished to ForgeBox
